Makefile

```diff
@@ -66,11 +66,11 @@ endif
 
 $(NODE_EXE): config.gypi out/Makefile
 	$(MAKE) -C out BUILDTYPE=Release V=$(V)
-	ln -fs out/Release/$(NODE_EXE) $@
+	if [ ! -r $(NODE_EXE) ]; then ln -fs out/Release/$(NODE_EXE) $@; fi
```

Comment:
Yep I'll add a comment - that's a fair point.
Regarding -L, to be honest I wasn't certain it was standard sh syntax, although it appears to work on all the systems I've tried (AIX, Solaris and of course linux) so I can add that in along with the -r (Since -L in itself will pass if it's a symlink that points to nothing)